desktop/package/linux/Haveno.AppDir/Haveno.desktop

```diff
@@ -0,0 +1,15 @@
+# If you want Haveno to appear in a Linux app launcher ("start menu"), install this by doing:
+# sudo desktop-file-install electrum-ltc.desktop
+
+[Desktop Entry]
+Comment=A decentralized monero exchange network.
+Exec=sh -c "PATH=\"\\$HOME/.local/bin:\\$PATH\"; bin/Haveno %u"
+GenericName[en_US]=Monero Exchange
+GenericName=Monero Exchange
+Icon=haveno
```

Comment:
This is just a question but is there any reason you chose to use `haveno` instead of a reverse-dns name like`exchange.haveno.Haveno`? If you must, it's fine, but the latter is needed for building Flatpaks and having two different appstream and icon files would be a pain to manage.

Reply:
No specific reason. I'm going to add a commit using `exchange.haven.Haveno`.

Reply:
Where specifically would you like me to change to using `exchange.haveno.Haveno`. It would be possible to rename the entire a `AppImage` to `exchange.haveno.Haveno` but the name entry should probably stay as `Haveno`.

Reply:
It would also be possible to keep the current name and just use symlinks.

Reply:
> Where specifically would you like me to change to using `exchange.haveno.Haveno`?
The `.desktop` spec https://specifications.freedesktop.org/desktop-entry-spec/1.3/file-naming.html says that the file name should follow this. I don't see anything about the icon, but it would be nice to rename/link the icon too since the Flatpak will need to use it.

Reply:
I'm fixing this now. Won't be able to do the symlink until I'm back on my Linux box but I'll fix everything else.